### PR TITLE
Finn Francis - We Got Coders groundwork

### PR DESCRIPTION
**Pull request by finn-francis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 ![We Got Coders Logo](http://wegotcoders.com/assets/logos/WEGOTCODERS_color.png "We Got Coders")
 
+
+#Some notes on my programming challenge
+
+I wasn't sure quite how to deal with this app that you sent me, I found working on the individual languages in a console window fairly easy, but this totally threw me.
+
+As I wasn't sure how to do a lot of this properly, so I decided to do what I could, and therefore demonstrate some of my programming knowlege.
```
Comment (dmgarland) [inline README.md]: I think this is a good approach :+1:

File: README.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
 ![We Got Coders Logo](http://wegotcoders.com/assets/logos/WEGOTCODERS_color.png "We Got Coders")
 
+
+#Some notes on my programming challenge
+
+I wasn't sure quite how to deal with this app that you sent me, I found working on the individual languages in a console window fairly easy, but this totally threw me.
+
+As I wasn't sure how to do a lot of this properly, so I decided to do what I could, and therefore demonstrate some of my programming knowlege.  
+
+For example I'm not sure where the content I was supposed to be laying out in main.erb was located, or how to arrange it if I couldn't see it. So I created a new page with my profile information on it.
```
Comment (dmgarland) [inline README.md]: In the real world we'll be on hand to ask; in this case I'm looking for your thought process and whether you can use your initiative. I'll take a look and let you know what I think!

File: primes.rb [hunk 3]
```diff
@@ -1,6 +1,60 @@
...
-  end
+  
+	# I understand how to make this run with input from the user throught the terminal (limit = gets.chomp)
+	# But I couldn't succesfull get it to work with the .erb files
+
+	require 'prime'
+
+	def self.sum_to(limit = 100)
+		sum = 0
+		Prime.each(limit) do |prime|
```
Comment (dmgarland) [inline primes.rb]: This works but also loses the opportunity to play around with some good prime number algorithms

File: primes.rb [hunk 4]
```diff
@@ -1,6 +1,60 @@
...
+	# 				break
+	# 			end
+	# 		end
+
+	# 		if primes[primes.length - 1] < i && !composite
+	# 				primes << i	
+	# 		end
+	# 	end
+	# 	return primes
+	# end
```
Comment (dmgarland) [inline primes.rb]: OK cool you had a go at it, what was the problem you ran into?

Comment (finn-francis) [inline primes.rb]: I wrote the algorithm in a separate file and it works fine, but when I moved it to primes.rb it threw up this error:

Internal Server Error
undefined method `join' for #<String:0x007fe2aa8649f0>
WEBrick/1.3.1 (Ruby/2.2.1/2015-02-26) at localhost:4567

**Pull request closed without merging**